ccvm_simulators/post_processor/tests/test_lbfgs.py

```diff
@@ -37,6 +37,22 @@ def test_postprocess_valid(self):
         error_message = "post_processing time must be greater than 0"
         self.assertGreater(self.post_processor.pp_time, 0, error_message)
 
+    def test_postprocess_default_upper_lower_clamp(self):
```

Comment:
Don't forget to rename the `test_postprocess_valid` to `test_postprocess_default_values` in this file, too :)

Reply:
Done.